File: ProTip365/Settings/SettingsLocalization.swift
import Foundation

// MARK: - Settings Localization

struct SettingsLocalization {
    let language: String

    init(language: String = "en") {
        self.language = language
    }

    // MARK: - General Settings

    var settingsTitle: String {
        switch language {
        case "fr": return "Réglages"
        case "es": return "Ajustes"
        default: return "Settings"
        }
    }

    var saveButton: String {
        switch language {
        case "fr": return "Sauvegarder"
        case "es": return "Guardar"
        default: return "Save"
        }
    }

    var savingButton: String {
        switch language {
        case "fr": return "Sauvegarde..."
        case "es": return "Guardando..."
        default: return "Saving..."
        }
    }

    var cancelButton: String {
        switch language {
        case "fr": return "Annuler"
        case "es": return "Cancelar"
        default: return "Cancel"
        }
    }

    var doneButton: String {
        switch language {
        case "fr": return "Terminé"
        case "es": return "Listo"
        default: return "Done"
        }
    }

    // MARK: - Profile Section

    var profileSection: String {
        switch language {
        case "fr": return "Profil"
        case "es": return "Perfil"
        default: return "Profile"
        }
    }

    var nameLabel: String {
        switch language {
        case "fr": return "Nom"
        case "es": return "Nombre"
        default: return "Name"
        }
    }

    var emailLabel: String {
        switch language {
        case "fr": return "Courriel"
        case "es": return "Correo electrónico"
        default: return "Email"
        }
    }

    var defaultHourlyRate: String {
        switch language {
        case "fr": return "Taux horaire par défaut"
        case "es": return "Tarifa por hora predeterminada"
        default: return "Default Hourly Rate"
        }
    }

    var tipTargetPercentage: String {
        switch language {
        case "fr": return "Objectif de pourboire (%)"
        case "es": return "Objetivo de propinas (%)"
        default: return "Tip Target Percentage (%)"
        }
    }

    var weekStartDay: String {
        switch language {
        case "fr": return "Début de semaine"
        case "es": return "Inicio de semana"
        default: return "Week Start Day"
        }
    }

    var useMultipleEmployers: String {
        switch language {
        case "fr": return "Utiliser plusieurs employeurs"
        case "es": return "Usar múltiples empleadores"
        default: return "Use Multiple Employers"
        }
    }

    var defaultEmployer: String {
        switch language {
        case "fr": return "Employeur par défaut"
        case "es": return "Empleador predeterminado"
        default: return "Default Employer"
        }
    }

    var defaultAlertLabel: String {
        switch language {
        case "fr": return "Alerte de quart par défaut"
        case "es": return "Alerta de turno predeterminada"
        default: return "Default Shift Alert"
        }
    }

    // MARK: - Targets Section

    var targetsSection: String {
        switch language {
        case "fr": return "Objectifs"
        case "es": return "Objetivos"
        default: return "Targets"
        }
    }

    var dailyTargets: String {
        switch language {
        case "fr": return "Objectifs quotidiens"
        case "es": return "Objetivos diarios"
        default: return "Daily Targets"
        }
    }

    var weeklyTargets: String {
        switch language {
        case "fr": return "Objectifs hebdomadaires"
        case "es": return "Objetivos semanales"
        default: return "Weekly Targets"
        }
    }

    var monthlyTargets: String {
        switch language {
        case "fr": return "Objectifs mensuels"
        case "es": return "Objetivos mensuales"
        default: return "Monthly Targets"
        }
    }

    var targetSales: String {
        switch language {
        case "fr": return "Ventes"
        case "es": return "Ventas"
        default: return "Sales"
        }
    }

    var targetHours: String {
        switch language {
        case "fr": return "Heures"
        case "es": return "Horas"
        default: return "Hours"
        }
    }

    // MARK: - Security Section

    var securitySection: String {
        switch language {
        case "fr": return "Sécurité"
        case "es": return "Seguridad"
        default: return "Security"
        }
    }

    var appLockSecurity: String {
        switch language {
        case "fr": return "Verrouillage d'application"
        case "es": return "Bloqueo de aplicación"
        default: return "App Lock Security"
        }
    }

    var noneSecurityType: String {
        switch language {
        case "fr": return "Aucun"
        case "es": return "Ninguno"
        default: return "None"
        }
    }

    var pinSecurityType: String {
        switch language {
        case "fr": return "Code PIN"
        case "es": return "Código PIN"
        default: return "PIN Code"
        }
    }

    var biometricSecurityType: String {
        switch language {
        case "fr": return "Biométrie"
        case "es": return "Biométrica"
        default: return "Biometric"
        }
    }

    var changePIN: String {
        switch language {
        case "fr": return "Changer le code PIN"
        case "es": return "Cambiar código PIN"
        default: return "Change PIN"
        }
    }

    // MARK: - Support Section

    var supportSection: String {
        switch language {
        case "fr": return "Support"
        case "es": return "Soporte"
        default: return "Support"
        }
    }

    var suggestIdeas: String {
        switch language {
        case "fr": return "Suggérer des idées"
        case "es": return "Sugerir ideas"
        default: return "Suggest Ideas"
        }
    }

    var exportData: String {
        switch language {
        case "fr": return "Exporter les données"
        case "es": return "Exportar datos"
        default: return "Export Data"
        }
    }

    var exportDataDescription: String {
        switch language {
        case "fr": return "CSV et autres formats"
        case "es": return "CSV y otros formatos"
        default: return "CSV and other formats"
        }
    }

    var onboardingGuide: String {
        switch language {
        case "fr": return "Guide de configuration"
        case "es": return "Guía de configuración"
        default: return "Setup Guide"
        }
    }

    var onboardingGuideSubtitle: String {
        switch language {
        case "fr": return "Revoir la configuration initiale"
        case "es": return "Revisar la configuración inicial"
        default: return "Review initial setup"
        }
    }

    var appTutorial: String {
        switch language {
        case "fr": return "Tutoriel de l'application"
        case "es": return "Tutorial de la aplicación"
        default: return "App Tutorial"
        }
    }

    // MARK: - Account Section

    var accountSection: String {
        switch language {
        case "fr": return "Compte"
        case "es": return "Cuenta"
        default: return "Account"
        }
    }

    var manageSubscription: String {
        switch language {
        case "fr": return "Gérer l'abonnement"
        case "es": return "Gestionar suscripción"
        default: return "Manage Subscription"
        }
    }

    var signOut: String {
        switch language {
        case "fr": return "Se déconnecter"
        case "es": return "Cerrar sesión"
        default: return "Sign Out"
        }
    }

    var deleteAccount: String {
        switch language {
        case "fr": return "Supprimer le compte"
        case "es": return "Eliminar cuenta"
        default: return "Delete Account"
        }
    }

    // MARK: - Work Defaults Section

    var workDefaultsSection: String {
        switch language {
        case "fr": return "Valeurs par défaut"
        case "es": return "Valores predeterminados"
        default: return "Work Defaults"
        }
    }

    var hourlyRateLabel: String {
        switch language {
        case "fr": return "Taux horaire"
        case "es": return "Tarifa por hora"
        default: return "Hourly Rate"
        }
    }

    var averageDeductionLabel: String {
        switch language {
        case "fr": return "Déductions AVG"
        case "es": return "Deducciones AVG"
        default: return "AVG deductions"
        }
    }

    var averageDeductionNoteTitle: String {
        switch language {
        case "fr": return "À propos des déductions"
        case "es": return "Acerca de las deducciones"
        default: return "About deductions"
        }
    }

    var averageDeductionNoteMessage: String {
        switch language {
        case "fr": return "Il s'agit de la déduction moyenne attendue sur votre salaire brut (impôts sur le revenu, cotisations sociales, etc.). Nous utiliserons ce pourcentage pour estimer votre salaire net, bien que le montant réel proviendra de votre bulletin de paie."
        case "es": return "Este es el porcentaje de deducción promedio esperado de su salario bruto (impuestos sobre la renta, seguridad social, etc.). Usaremos esto para estimar su salario neto, aunque el monto real vendrá de su nómina."
        default: return "This is the expected average deduction on your gross salary (income taxes, social security, etc.). We'll use this to estimate your net salary, though the actual amount will come from your payroll."
        }
    }

    var variableScheduleLabel: String {
        switch language {
        case "fr": return "Horaire variable"
        case "es": return "Horario variable"
        default: return "Variable Schedule"
        }
    }

    var variableScheduleDescription: String {
        switch language {
        case "fr": return "J'ai un horaire de travail variable"
        case "es": return "Tengo un horario de trabajo variable"
        default: return "I have a variable work schedule"
        }
    }

    var variableScheduleEnabledTitle: String {
        switch language {
        case "fr": return "Mode horaire variable activé"
        case "es": return "Modo horario variable activado"
        default: return "Variable Schedule Mode Active"
        }
    }

    var variableScheduleEnabledMessage: String {
        switch language {
        case "fr": return "Les objectifs hebdomadaires et mensuels pour les ventes et les heures sont désactivés. Seuls les objectifs quotidiens sont visibles car ils sont plus pertinents pour les horaires variables."
        case "es": return "Los objetivos semanales y mensuales para ventas y horas están desactivados. Solo los objetivos diarios son visibles porque son más relevantes para horarios variables."
        default: return "Weekly and monthly targets for sales and hours are hidden. Only daily targets are shown as they're more relevant for variable schedules."
        }
    }

    var variableScheduleNoteTitle: String {
        switch language {
        case "fr": return "Horaire variable"
        case "es": return "Horario variable"
        default: return "Variable Schedule"
        }
    }

    var variableScheduleNoteMessage: String {
        switch language {
        case "fr": return "Si vous avez un horaire variable, vous pouvez utiliser uniquement les objectifs quotidiens, qui seront plus pertinents. Les objectifs hebdomadaires et mensuels ne sont pas significatifs si vous ne travaillez pas un nombre fixe de jours."
        case "es": return "Si tienes un horario variable, puedes usar solo los objetivos diarios, que serán más relevantes. Los objetivos semanales y mensuales no tienen sentido si no trabajas un número fijo de días."
        default: return "If you have a variable schedule, you can just use daily targets, which will be more relevant. Weekly and monthly targets don't make sense if you don't work a fixed number of days."
        }
    }

    var variableScheduleHoursNoteMessage: String {
        switch language {
        case "fr": return "Si vous avez un horaire variable, vous pouvez utiliser uniquement les objectifs d'heures quotidiens, qui seront plus pertinents. Les objectifs hebdomadaires et mensuels ne sont pas significatifs si vous ne travaillez pas un nombre fixe de jours."
        case "es": return "Si tienes un horario variable, puedes usar solo los objetivos de horas diarios, que serán más relevantes. Los objetivos semanales y mensuales no tienen sentido si no trabajas un número fijo de días."
        default: return "If you have a variable schedule, you can just use daily hours targets, which will be more relevant. Weekly and monthly targets don't make sense if you don't work a fixed number of days."
        }
    }

    // MARK: - Targets Section

    var yourTargetsTitle: String {
        switch language {
        case "fr": return "Vos objectifs"
        case "es": return "Tus objetivos"
        default: return "Your Targets"
        }
    }

    var tipTargetsSection: String {
        switch language {
        case "fr": return "Objectifs de pourboires"
        case "es": return "Objetivos de propinas"
        default: return "Tip Targets"
        }
    }

    var tipPercentageTargetLabel: String {
        switch language {
        case "fr": return "Pourcentage de pourboire cible"
        case "es": return "Porcentaje objetivo de propinas"
        default: return "Target Tip Percentage"
        }
    }

    var tipPercentageShortLabel: String {
        switch language {
        case "fr": return "Pourboire %"
        case "es": return "Propina %"
        default: return "Tip %"
        }
    }

    var tipPercentageNoteTitle: String {
        switch language {
        case "fr": return "Pourcentage des ventes"
        case "es": return "Porcentaje de ventas"
        default: return "Percentage of Sales"
        }
    }

    var tipPercentageNoteMessage: String {
        switch language {
        case "fr": return "Définissez votre objectif de pourboire en pourcentage de vos ventes totales. Par exemple, 15% signifie que vous visez 15% de vos ventes en pourboires."
        case "es": return "Establezca su objetivo de propinas como porcentaje de sus ventas totales. Por ejemplo, 15% significa que apunta a 15% de sus ventas en propinas."
        default: return "Set your tip goal as a percentage of your total sales. For example, 15% means you aim for 15% of your sales in tips."
        }
    }

    var salesTargetsSection: String {
        switch language {
        case "fr": return "Objectifs de ventes"
        case "es": return "Objetivos de ventas"
        default: return "Sales Targets"
        }
    }

    var hoursTargetsSection: String {
        switch language {
        case "fr": return "Objectifs d'heures"
        case "es": return "Objetivos de horas"
        default: return "Hours Targets"
        }
    }

    var dailySalesTargetLabel: String {
        switch language {
        case "fr": return "Ventes quotidiennes"
        case "es": return "Ventas diarias"
        default: return "Daily Sales"
        }
    }

    var weeklySalesTargetLabel: String {
        switch language {
        case "fr": return "Ventes hebdomadaires"
        case "es": return "Ventas semanales"
        default: return "Weekly Sales"
        }
    }

    var monthlySalesTargetLabel: String {
        switch language {
        case "fr": return "Ventes mensuelles"
        case "es": return "Ventas mensuales"
        default: return "Monthly Sales"
        }
    }

    var dailyHoursTargetLabel: String {
        switch language {
        case "fr": return "Heures quotidiennes"
        case "es": return "Horas diarias"
        default: return "Daily Hours"
        }
    }

    var weeklyHoursTargetLabel: String {
        switch language {
        case "fr": return "Heures hebdomadaires"
        case "es": return "Horas semanales"
        default: return "Weekly Hours"
        }
    }

    var monthlyHoursTargetLabel: String {
        switch language {
        case "fr": return "Heures mensuelles"
        case "es": return "Horas mensuales"
        default: return "Monthly Hours"
        }
    }

    var setYourGoalsTitle: String {
        switch language {
        case "fr": return "🎯 Fixez vos objectifs"
        case "es": return "🎯 Establece tus metas"
        default: return "🎯 Set your goals"
        }
    }

    var setYourGoalsDescription: String {
        switch language {
        case "fr": return "Définissez des objectifs quotidiens, hebdomadaires et mensuels. Votre tableau de bord affichera les progrès comme '2.0/8h' pour les heures travaillées vs prévues."
        case "es": return "Establezca objetivos diarios, semanales y mensuales. Su panel mostrará el progreso como '2.0/8h' para horas trabajadas vs esperadas."
        default: return "Set daily, weekly, and monthly targets. Your dashboard will show progress like '2.0/8h' for hours worked vs expected."
        }
    }

    // MARK: - Language Section

    var languageSection: String {
        switch language {
        case "fr": return "Langue"
        case "es": return "Idioma"
        default: return "Language"
        }
    }

    var languageLabel: String {
        switch language {
        case "fr": return "Choisir la langue"
        case "es": return "Elegir idioma"
        default: return "Choose Language"
        }
    }

    // MARK: - App Info Section

    var howToUseText: String {
        switch language {
        case "fr": return "Comment utiliser ProTip365"
        case "es": return "Cómo usar ProTip365"
        default: return "How to Use ProTip365"
        }
    }

    // MARK: - Subscription Section

    var cancelSubscriptionTitle: String {
        switch language {
        case "fr": return "Annuler l'abonnement"
        case "es": return "Cancelar suscripción"
        default: return "Cancel Subscription"
        }
    }

    var cancelSubscriptionInstructions: String {
        switch language {
        case "fr": return "Allez dans votre compte Apple pour gérer votre abonnement. Puisque l'abonnement est géré par Apple, vous devez l'annuler directement dans vos paramètres Apple."
        case "es": return "Ve a tu cuenta de Apple para gestionar tu suscripción. Como la suscripción es gestionada por Apple, debes cancelarla directamente en tu configuración de Apple."
        default: return "Go to your Apple account to manage your subscription. Since the subscription is managed by Apple, you must cancel it directly in your Apple settings."
        }
    }

    var cancelSubscriptionWarning: String {
        switch language {
        case "fr": return "⚠️ Important: Vous ne pourrez plus utiliser l'application après l'annulation. Pensez à exporter vos données avant de procéder."
        case "es": return "⚠️ Importante: No podrás usar la aplicación después de cancelar. Considera exportar tus datos antes de proceder."
        default: return "⚠️ Important: You will not be able to use the app after canceling. Consider exporting your data before proceeding."
        }
    }

    var goToAppleAccountText: String {
        switch language {
        case "fr": return "Aller aux paramètres Apple"
        case "es": return "Ir a configuración de Apple"
        default: return "Go to Apple Settings"
        }
    }

    // MARK: - Suggestion Section

    var suggestionEmailPlaceholder: String {
        switch language {
        case "fr": return "Votre adresse email"
        case "es": return "Su dirección de correo"
        default: return "Your email address"
        }
    }

    var yourSuggestionHeader: String {
        switch language {
        case "fr": return "Votre suggestion"
        case "es": return "Su sugerencia"
        default: return "Your suggestion"
        }
    }

    var suggestionFooter: String {
        switch language {
        case "fr": return "Partagez vos idées pour améliorer ProTip365"
        case "es": return "Comparta sus ideas para mejorar ProTip365"
        default: return "Share your ideas to improve ProTip365"
        }
    }

    var sendSuggestionButton: String {
        switch language {
        case "fr": return "Envoyer"
        case "es": return "Enviar"
        default: return "Send"
        }
    }

    var thankYouTitle: String {
        switch language {
        case "fr": return "Merci!"
        case "es": return "¡Gracias!"
        default: return "Thank You!"
        }
    }

    var thankYouMessage: String {
        switch language {
        case "fr": return "Merci pour vos commentaires, nous l'apprécions."
        case "es": return "Gracias por sus comentarios, lo apreciamos."
        default: return "Thank you for your feedback, we appreciate it."
        }
    }

    var contactSupport: String {
        switch language {
        case "fr": return "Contacter le support"
        case "es": return "Contactar soporte"
        default: return "Contact Support"
        }
    }

    var supportDescription: String {
        switch language {
        case "fr": return "Décrivez votre problème et nous vous répondrons dans les plus brefs délais."
        case "es": return "Describa su problema y le responderemos lo antes posible."
        default: return "Describe your issue and we'll get back to you as soon as possible."
        }
    }

    var yourMessage: String {
        switch language {
        case "fr": return "Votre message"
        case "es": return "Su mensaje"
        default: return "Your message"
        }
    }

    var describeYourIssue: String {
        switch language {
        case "fr": return "Décrivez votre problème..."
        case "es": return "Describa su problema..."
        default: return "Describe your issue..."
        }
    }

    var writeYourSuggestion: String {
        switch language {
        case "fr": return "Écrivez votre suggestion..."
        case "es": return "Escriba su sugerencia..."
        default: return "Write your suggestion..."
        }
    }

    // MARK: - Common Labels

    var noneLabel: String {
        switch language {
        case "fr": return "Aucun"
        case "es": return "Ninguno"
        default: return "None"
        }
    }

    var saveSettingsText: String {
        switch language {
        case "fr": return "Enregistrer"
        case "es": return "Guardar"
        default: return "Save"
        }
    }

    var savedText: String {
        switch language {
        case "fr": return "Enregistré!"
        case "es": return "¡Guardado!"
        default: return "Saved!"
        }
    }

    var discardButton: String {
        switch language {
        case "fr": return "Ignorer"
        case "es": return "Descartar"
        default: return "Discard"
        }
    }

    // MARK: - Messages

    var unsavedChangesTitle: String {
        switch language {
        case "fr": return "Modifications non sauvegardées"
        case "es": return "Cambios no guardados"
        default: return "Unsaved Changes"
        }
    }

    var unsavedChangesMessage: String {
        switch language {
        case "fr": return "Vous avez des modifications non sauvegardées. Voulez-vous les sauvegarder avant de continuer?"
        case "es": return "Tienes cambios sin guardar. ¿Quieres guardarlos antes de continuar?"
        default: return "You have unsaved changes. Would you like to save them before continuing?"
        }
    }

    var settingsUpdated: String {
        switch language {
        case "fr": return "Réglages mis à jour!"
        case "es": return "¡Ajustes actualizados!"
        default: return "Settings Updated!"
        }
    }

    var errorSavingSettings: String {
        switch language {
        case "fr": return "Erreur lors de la sauvegarde"
        case "es": return "Error al guardar"
        default: return "Error saving settings"
        }
    }

    // MARK: - Alert Messages

    var signOutConfirmTitle: String {
        switch language {
        case "fr": return "Confirmer"
        case "es": return "Confirmar"
        default: return "Confirm"
        }
    }

    var signOutConfirmMessage: String {
        switch language {
        case "fr": return "Voulez-vous vraiment vous déconnecter?"
        case "es": return "¿Realmente quieres cerrar sesión?"
        default: return "Are you sure you want to sign out?"
        }
    }

    var deleteAccountConfirmTitle: String {
        switch language {
        case "fr": return "Supprimer le compte"
        case "es": return "Eliminar cuenta"
        default: return "Delete Account"
        }
    }

    var deleteAccountConfirmMessage: String {
        switch language {
        case "fr": return "Êtes-vous sûr de vouloir supprimer votre compte? Cette action est irréversible.\n\nRappel: Vous devez également aller dans votre compte Apple pour annuler votre abonnement."
        case "es": return "¿Está seguro de que desea eliminar su cuenta? Esta acción es irreversible.\n\nRecordatorio: También debe ir a su cuenta de Apple para cancelar su suscripción."
        default: return "Are you sure you want to delete your account? This action cannot be undone.\n\nReminder: You must also go to your Apple account to cancel your subscription."
        }
    }
}

File: _Backups/2025-12-08_Pre_Optimization/ProTip365Android/app/src/main/java/com/protip365/app/presentation/localization/SettingsLocalization.kt
package com.protip365.app.presentation.localization

class SettingsLocalization(private val language: String) {

    // MARK: - General Settings
    val settingsTitle: String
        get() = when (language) {
            "fr" -> "Réglages"
            "es" -> "Ajustes"
            else -> "Settings"
        }

    val saveButton: String
        get() = when (language) {
            "fr" -> "Sauvegarder"
            "es" -> "Guardar"
            else -> "Save"
        }

    val savingButton: String
        get() = when (language) {
            "fr" -> "Sauvegarde..."
            "es" -> "Guardando..."
            else -> "Saving..."
        }

    val cancelButton: String
        get() = when (language) {
            "fr" -> "Annuler"
            "es" -> "Cancelar"
            else -> "Cancel"
        }

    val doneButton: String
        get() = when (language) {
            "fr" -> "Terminé"
            "es" -> "Listo"
            else -> "Done"
        }

    // MARK: - Sections
    val profileSection: String
        get() = when (language) {
            "fr" -> "Profil"
            "es" -> "Perfil"
            else -> "Profile"
        }

    val workDefaultsSection: String
        get() = when (language) {
            "fr" -> "Défauts de travail"
            "es" -> "Configuración de trabajo"
            else -> "Work Defaults"
        }

    val securitySection: String
        get() = when (language) {
            "fr" -> "Sécurité"
            "es" -> "Seguridad"
            else -> "Security"
        }

    val subscriptionSection: String
        get() = when (language) {
            "fr" -> "Abonnement"
            "es" -> "Suscripción"
            else -> "Subscription"
        }

    val supportSection: String
        get() = when (language) {
            "fr" -> "Support"
            "es" -> "Soporte"
            else -> "Support"
        }

    val aboutSection: String
        get() = when (language) {
            "fr" -> "À propos"
            "es" -> "Acerca de"
            else -> "About"
        }

    // MARK: - Alert Settings
    val alertLabel: String
        get() = when (language) {
            "fr" -> "Alerte"
            "es" -> "Alerta"
            else -> "Alert"
        }

    val alertNone: String
        get() = when (language) {
            "fr" -> "Aucune"
            "es" -> "Ninguna"
            else -> "None"
        }

    val alert15Minutes: String
        get() = when (language) {
            "fr" -> "15 minutes avant"
            "es" -> "15 minutos antes"
            else -> "15 minutes before"
        }

    val alert30Minutes: String
        get() = when (language) {
            "fr" -> "30 minutes avant"
            "es" -> "30 minutos antes"
            else -> "30 minutes before"
        }

    val alert1Hour: String
        get() = when (language) {
            "fr" -> "1 heure avant"
            "es" -> "1 hora antes"
            else -> "1 hour before"
        }

    val alert1Day: String
        get() = when (language) {
            "fr" -> "1 jour avant"
            "es" -> "1 día antes"
            else -> "1 day before"
        }

    val defaultAlertLabel: String
        get() = when (language) {
            "fr" -> "Alerte par défaut"
            "es" -> "Alerta predeterminada"
            else -> "Default Alert"
        }

    // MARK: - Profile Settings
    val setNameText: String
        get() = when (language) {
            "fr" -> "Définir votre nom"
            "es" -> "Establecer tu nombre"
            else -> "Set your name"
        }

    val tapToSetProfileText: String
        get() = when (language) {
            "fr" -> "Appuyez pour définir votre profil"
            "es" -> "Toca para establecer tu perfil"
            else -> "Tap to set your profile"
        }

    val tapToUpdateProfileText: String
        get() = when (language) {
            "fr" -> "Appuyez pour mettre à jour le profil"
            "es" -> "Toca para actualizar el perfil"
            else -> "Tap to update profile"
        }

    val changePasswordText: String
        get() = when (language) {
            "fr" -> "Changer le mot de passe"
            "es" -> "Cambiar contraseña"
            else -> "Change Password"
        }

    val updateAccountPasswordText: String
        get() = when (language) {
            "fr" -> "Mettre à jour le mot de passe de votre compte"
            "es" -> "Actualizar la contraseña de tu cuenta"
            else -> "Update your account password"
        }

    // MARK: - Work Defaults
    val defaultHourlyRateText: String
        get() = when (language) {
            "fr" -> "Taux horaire par défaut"
            "es" -> "Tarifa por hora predeterminada"
            else -> "Default Hourly Rate"
        }

    val multipleEmployersText: String
        get() = when (language) {
            "fr" -> "Employeurs multiples"
            "es" -> "Empleadores múltiples"
            else -> "Multiple Employers"
        }

    val enabledText: String
        get() = when (language) {
            "fr" -> "Activé"
            "es" -> "Habilitado"
            else -> "Enabled"
        }

    val disabledText: String
        get() = when (language) {
            "fr" -> "Désactivé"
            "es" -> "Deshabilitado"
            else -> "Disabled"
        }

    val cashOutText: String
        get() = when (language) {
            "fr" -> "Encaissement"
            "es" -> "Cobro"
            else -> "Cash Out"
        }

    val trackingEnabledText: String
        get() = when (language) {
            "fr" -> "Suivi activé"
            "es" -> "Seguimiento habilitado"
            else -> "Tracking enabled"
        }

    val trackingDisabledText: String
        get() = when (language) {
            "fr" -> "Suivi désactivé"
            "es" -> "Seguimiento deshabilitado"
            else -> "Tracking disabled"
        }

    // MARK: - Security
    val pinProtectionText: String
        get() = when (language) {
            "fr" -> "Protection par code PIN"
            "es" -> "Protección con PIN"
            else -> "PIN Protection"
        }

    val biometricAuthenticationText: String
        get() = when (language) {
            "fr" -> "Authentification biométrique"
            "es" -> "Autenticación biométrica"
            else -> "Biometric Authentication"
        }

    // MARK: - Subscription
    val fullAccessText: String
        get() = when (language) {
            "fr" -> "Accès complet"
            "es" -> "Acceso completo"
            else -> "Full Access"
        }

    val partTimeProText: String
        get() = when (language) {
            "fr" -> "Pro à temps partiel"
            "es" -> "Pro de medio tiempo"
            else -> "Part-Time Pro"
        }

    val freeTrialText: String
        get() = when (language) {
            "fr" -> "Essai gratuit"
            "es" -> "Prueba gratuita"
            else -> "Free Trial"
        }

    val unlimitedShiftsEntriesText: String
        get() = when (language) {
            "fr" -> "Quarts et entrées illimités"
            "es" -> "Turnos y entradas ilimitados"
            else -> "Unlimited shifts & entries"
        }

    val limitedFeaturesText: String
        get() = when (language) {
            "fr" -> "Fonctionnalités limitées"
            "es" -> "Características limitadas"
            else -> "Limited features"
        }

    val upgradeToFullAccessText: String
        get() = when (language) {
            "fr" -> "Passer à l'accès complet"
            "es" -> "Actualizar a acceso completo"
            else -> "Upgrade to Full Access"
        }

    val unlimitedEverythingText: String
        get() = when (language) {
            "fr" -> "Tout illimité"
            "es" -> "Todo ilimitado"
            else -> "Unlimited everything"
        }

    // MARK: - Support
    val helpCenterText: String
        get() = when (language) {
            "fr" -> "Centre d'aide"
            "es" -> "Centro de ayuda"
            else -> "Help Center"
        }

    val viewGuidesTutorialsText: String
        get() = when (language) {
            "fr" -> "Voir les guides et tutoriels"
            "es" -> "Ver guías y tutoriales"
            else -> "View guides and tutorials"
        }

    val contactSupportText: String
        get() = when (language) {
            "fr" -> "Contacter le support"
            "es" -> "Contactar soporte"
            else -> "Contact Support"
        }

    val getHelpFromTeamText: String
        get() = when (language) {
            "fr" -> "Obtenir de l'aide de notre équipe"
            "es" -> "Obtener ayuda de nuestro equipo"
            else -> "Get help from our team"
        }

    val privacyPolicyText: String
        get() = when (language) {
            "fr" -> "Politique de confidentialité"
            "es" -> "Política de privacidad"
            else -> "Privacy Policy"
        }

    val viewPrivacyPolicyText: String
        get() = when (language) {
            "fr" -> "Voir notre politique de confidentialité"
            "es" -> "Ver nuestra política de privacidad"
            else -> "View our privacy policy"
        }

    val termsOfServiceText: String
        get() = when (language) {
            "fr" -> "Conditions d'utilisation"
            "es" -> "Términos de servicio"
            else -> "Terms of Service"
        }

    val viewTermsConditionsText: String
        get() = when (language) {
            "fr" -> "Voir les termes et conditions"
            "es" -> "Ver términos y condiciones"
            else -> "View terms and conditions"
        }

    // MARK: - Account Actions
    val signOutText: String
        get() = when (language) {
            "fr" -> "Se déconnecter"
            "es" -> "Cerrar sesión"
            else -> "Sign Out"
        }

    val signOutAccountText: String
        get() = when (language) {
            "fr" -> "Se déconnecter de votre compte"
            "es" -> "Cerrar sesión de tu cuenta"
            else -> "Sign out of your account"
        }

    val deleteAccountText: String
        get() = when (language) {
            "fr" -> "Supprimer le compte"
            "es" -> "Eliminar cuenta"
            else -> "Delete Account"
        }

    val permanentlyDeleteAccountText: String
        get() = when (language) {
            "fr" -> "Supprimer définitivement votre compte"
            "es" -> "Eliminar permanentemente tu cuenta"
            else -> "Permanently delete your account"
        }

    // MARK: - App Settings
    val appSettingsText: String
        get() = when (language) {
            "fr" -> "Paramètres de l'application"
            "es" -> "Configuración de la aplicación"
            else -> "App Settings"
        }

    val languageText: String
        get() = when (language) {
            "fr" -> "Langue"
            "es" -> "Idioma"
            else -> "Language"
        }

    val choosePreferredLanguageText: String
        get() = when (language) {
            "fr" -> "Choisissez votre langue préférée"
            "es" -> "Elige tu idioma preferido"
            else -> "Choose your preferred language"
        }

    // MARK: - About
    val versionText: String
        get() = when (language) {
            "fr" -> "Version"
            "es" -> "Versión"
            else -> "Version"
        }

    // MARK: - Variable Schedule
    val variableScheduleLabel: String
        get() = when (language) {
            "fr" -> "J'ai un horaire de travail variable"
            "es" -> "Tengo un horario de trabajo variable"
            else -> "I have a variable work schedule"
        }

    val variableScheduleDescription: String
        get() = when (language) {
            "fr" -> "Les objectifs hebdomadaires et mensuels pour les ventes et les heures sont masqués. Seuls les objectifs quotidiens sont affichés car ils sont plus pertinents pour les horaires variables."
            "es" -> "Los objetivos semanales y mensuales para ventas y horas están ocultos. Solo se muestran los objetivos diarios ya que son más relevantes para horarios variables."
            else -> "Weekly and monthly targets for sales and hours are hidden. Only daily targets are shown as they're more relevant for variable schedules."
        }

    val variableScheduleEnabledTitle: String
        get() = when (language) {
            "fr" -> "Horaires variables activés"
            "es" -> "Horario variable habilitado"
            else -> "Variable schedule enabled"
        }

    val variableScheduleEnabledMessage: String
        get() = when (language) {
            "fr" -> "Si vous avez un horaire variable, vous pouvez simplement utiliser les objectifs quotidiens, qui seront plus pertinents. Les objectifs hebdomadaires et mensuels n'ont pas de sens si vous ne travaillez pas un nombre fixe de jours."
            "es" -> "Si tienes un horario variable, puedes simplemente usar objetivos diarios, que serán más relevantes. Los objetivos semanales y mensuales no tienen sentido si no trabajas un número fijo de días."
            else -> "If you have a variable schedule, you can just use daily targets, which will be more relevant. Weekly and monthly targets don't make sense if you don't work a fixed number of days."
        }
}
